lib/zeus/api_client/rest_interface.rb

```diff
@@ -21,20 +21,49 @@
 module RestInterface
   private
 
-  def get(path, params = {})
-    RestClient.get "#{@endpoint}#{path}", params: params
+  def get(path, token, bucket_name, params = {})
```

Comment:
@victorfonsec4 
Thanks for the suggestion, already implemented a `make_header` function in `lib/zeus/api_client/rest_client.rb`